README.rst

```diff
@@ -56,12 +56,8 @@
 |circleci| |travisci| |appveyor| |codecov| |codeclimate|
 
 EinsteinPy is an open source pure Python package dedicated to problems arising in 
-General Relativity and relativistic physics, such as goedesics calculation for vacuum 
-solutions for Einstein's field equations, calculation of various quantities in these 
-geometries like Schwarzschild Radius and event horizon. The library also has functions 
-for Symbolic calculations in GR like Christoffel Symbols and much more is planned. 
-The library aims to solve Einstein's field equations for arbitarily complicated 
-matter distribution as one of the main goals. 
+General Relativity and relativistic physics, such as geodesics calculation for vacuum solutions for Einstein's field equations, calculation of various quantities in these geometries like Schwarzschild Radius and the event horizon. The library also has functions for Symbolic calculations in GR like Christoffel Symbols and much more is planned. 
```

Comment:
We don't want to increase line length :/

Reply:
I have made the changes to revert to the original line length.